nautobot/core/tests/test_views.py

```diff
@@ -255,6 +255,25 @@ def test_installed_plugins_disabled_to_non_staff_user_with_hide_restricted_ui_fa
 
 
 class LoginUI(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.footer_links = """
```

Comment:
This seems fragile. Instead can we just use the `assertContains` to search for the strings we'd like to avoid?

Also, "New UI" does not exist in 1.6.

Reply:
Agreed, let's use some more narrowly targeted `assertNotContains`/`assertNotIn` checks instead.